Source/WebCore/Modules/applepay/ApplePayLogoSystemImage.h

```diff
@@ -41,7 +41,7 @@ enum class ApplePayLogoStyle : bool {
 };
 
 class WEBCORE_EXPORT ApplePayLogoSystemImage final : public SystemImage {
-    WTF_MAKE_TZONE_ALLOCATED(ApplePayLogoSystemImage);
+    WTF_MAKE_TZONE_ALLOCATED_EXPORT(ApplePayLogoSystemImage, WEBCORE_EXPORT);
```

Comment:
The "class WEBCORE_EXPORT" decl doesn't cause all the methods/operators to be exported already?  Does this fix some linker error you're encountering?

Reply:
For some of these classes I did get linking errors.  Given that these classes all have WEBCORE_EXPORT, the TZone annotations should match as other users of the WebCore framework may have dependencies on these exported classes.

Reply:
Yeah, they could have dependencies, but I would have thought that the class-level exporting would cause everything in the class to be exported. (Notice how nothing else is explicitly exported in any of these classes which seems to imply the methods get exported, otherwise it seems there'd be nothing you can do with these classes). But maybe not. If it fixes the linker errors, then okay.